app/views/ideas/_form.html.erb

```diff
@@ -102,3 +102,26 @@
     <% end %>
   </div>
 </div>
+<script>
+
+  var business_area = document.getElementById('business-area');
+  var area_of_interest = document.getElementById('idea_area_of_interest');
+  var it_system = document.getElementById('it-system');
+
+  business_area.style.display = 'none';
+  it_system.style.display = 'none';
+
+  area_of_interest.addEventListener('change', function() {
+   if (area_of_interest.value === 'other_business_area'){
+    business_area.style.display = "block"
+   }else {
+      business_area.style.display = "none"
+   }
+   if (area_of_interest.value === 'it_development'){
+     it_system.style.display = "block"
+   }else {
+      it_system.style.display = "none"
+   }
+  });
+
+</script>
```

Comment:
I don't think we should be storing this javascript in the ERB.  Have a look at the documents here about the Rais Asset Pipeline https://guides.rubyonrails.org/asset_pipeline.html#asset-organization in particular where to put and reference javascript.